aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -218,13 +221,32 @@ def __init__(
         cors: bool,
         compress: bool,
         cache_control: Optional[str],
+        middleware: Optional[List[Callable]],
```

Comment:
```suggestion
        middlewares: Optional[List[Callable]],
```

plural to signal the list of middlewares ;)